web/source/dom/domManager.ts

```diff
@@ -233,7 +241,6 @@ namespace com.keyman.dom {
 
       if(!this.isAttached(Pelem)) {
         this.setupElementAttachment(Pelem);
```

Comment:
This seems to be done again in `enableInputElement()`, called on line 247, albeit in a more nuanced manner (handling iframes)